mantidproject/mantid
# Update tools dialogs in tomographic reconstruction GUI for new versions

Fixes #12900.

Also fixes #12898 for now, with more up-to-date documentation.
Uses the newer scripts and versions of the tools introduced in #14708.

**To test**:
- Code review. There are still issues with the savu related code, but this is not under our control.
- The tests of ImageROIPresenter and TomographyIfacePresenter have been updated and should pass.
- The documentation should read acceptable, and the new screenshots should display correctly (note there are several warnings and TODOs and this is growing/work in progress).
- In the run tab, local runs won't work (unless you're on the special machine "imat analysis"). There's no proper error detection/reporting for this, and this needs fixing. Remote runs should work well regardless of your local machine (assuming you have credentials for SCARF). All other tabs should behave normally.
- For a simple remote job submission you can use the "custom" tool. Set the runnable to `/work/imat/phase_commissioning/scripts/Imaging/IMAT/tomo_reconstruct.py`, and options to: `--help`
  -Remote job submissions can be double-checked at https://portal.scarf.rl.ac.uk. For astra/tomopy submissions, the scripts should get to the poin that they report that no data could be loaded if you give an inexistent input path for sample images.
- Perform few steps of an interactive session. Example:
  - open a stack of images in the "ROI, etc." tab. There is an example (from ISIS): babylon5\Public\fedemp\test_stack_larmor_summed_101510. Play with the parameters.
  - In the "Visualize" tab, explore folders, and check that you get informative error messages if you try to send files to tools that are not setup.
  - In the "Run" tab submit a few different jobs (tools, custom, etc.). I can give more examples/details.
